ValetWatchTest Extension/InterfaceController.m

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
As for cocoapods, you'll need to change our `Gemfile` to use `0.39.0`, not the `0.37.2` it is currently using.